stdlib/gc.mli

```diff
@@ -79,7 +79,9 @@ type stat =
        are not available for allocation. *)
 
     compactions : int;
-    (** Number of heap compactions since the program was started. *)
+    (** Note: As of OCaml 5.0, compaction is not available and this field's
+        value will always be zero.
+        Number of heap compactions since the program was started. *)
```

Comment:
I would remove the last sentence - i.e. we just state that it's always zero because compaction was removed.